Mathlib/SetTheory/Game/PGame.lean

```diff
@@ -349,15 +349,15 @@ instance isEmpty_one_rightMoves : IsEmpty (RightMoves 1) :=
 
 /-- The less or equal relation on pre-games.
 
-If `0 ≤ x`, then Left can win `x` as the second player. -/
+If `0 ≤ x`, then Left can win `x` as the second player. `x ≤ y` means that `0 ≤ y - x`. -/
```

Comment:
You mention this isn't how it's actually defined - is there a lemma saying these are equivalent though? If so, let's reference it here.

Reply:
I've added that to the docstring.